apps/server/src/modules/tool/tool.module.ts

```diff
@@ -1,35 +1,43 @@
 import { forwardRef, Module } from '@nestjs/common';
+import { EncryptionModule } from '@shared/infra/encryption';
+import { OauthProviderServiceModule } from '@shared/infra/oauth-provider';
 import {
-	PseudonymsRepo,
-	ExternalToolRepoMapper,
-	ExternalToolRepo,
-	SchoolExternalToolRepo,
 	ContextExternalToolRepo,
+	ExternalToolRepo,
+	ExternalToolRepoMapper,
 	ExternalToolSortingMapper,
+	PseudonymsRepo,
+	SchoolExternalToolRepo,
 } from '@shared/repo';
 import { LoggerModule } from '@src/core/logger';
-import { OauthProviderServiceModule } from '@shared/infra/oauth-provider';
-import { EncryptionModule } from '@shared/infra/encryption';
+import { AuthorizationModule } from '../authorization';
```

Comment:
I think we should use relative imports for other modules @src/modules/authorization e.g.